packages/framework/client-logger/fluid-telemetry/README.md

```diff
@@ -0,0 +1,160 @@
+        console.log(event);
+    }
+}
+
+// 3. Next, we'll Create the telemetry config object.
+// Note that we have to obtain the containerId before we can do this.
+const telemetryConfig: TelemetryConfig = {
+    container: myAppContainer,
+    containerId: myAppContainerId,
+    consumers: [new MySimpleTelemetryConsumer(appInsightsClient)],
+};
+
+// 4. Start Telemetry
+startTelemetry(telemetryConfig);
+
+// Done! Your container telemetry is now being created and sent to your Telemetry Consumer
+```
+
+## Example 2: Logging container telemetry to Azure App Insights
+
+Before you can get telemetry sent to Azure App Insights, you'll need to create an Instance of App Insights on Azure. Then you'll be able to create an Azure App Insights client that you can easily turn into a ITelemetryConsumer and finally hook it up to container telemetry.
+
+### Step 1: First, we'll have to create our own telemetry consumer which extends the ITelemetryConsumer interface using our Azure App Insights client:
+
+```ts
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import { ITelemetryConsumer } from "@fluidframework/fluid-telemetry";
+
+class AppInsightsTelemetryConsumer implements ITelemetryConsumer {
+	constructor(private readonly appInsightsClient: ApplicationInsights) {}
+
+	consume(event: IFluidTelemetry) {
+		this.appInsightsClient.trackEvent({
+			name: event.eventName,
+			properties: event,
+		});
+	}
+}
+```
+
+#### Step 2: Now, let's start the telemetry production and hook in our telemetry consumer from step 1. We will be initializing our telemetry collection where we initialize our containers:
+
+```ts
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import { IFluidContainer } from "@fluidframework/fluid-static";
+import { ITelemetryConsumer , TelemetryConfig, startTelemetry, IFluidTelemetry } from "@fluidframework/external-telemetry"
+
+// 1: This is supposed to be your code for loading a Fluid Container
+let myAppContainer: IFluidContainer;
+let myAppContainerId: string;
+if (containerExists) {
+    myAppContainerId = {...your code to get the id of the existing container}
+    myAppContainer = {...your code to load a Fluid Container from myAppContainerId}
+} else {
+    myAppContainer = {...your code to create a new Fluid Container}
+    myAppContainerId = await myAppContainer.attach();
+}
+
+// 2: This is our implementation of ITelemetryConsumer that will send telemetry to Azure App Insights
+class AppInsightsTelemetryConsumer implements ITelemetryConsumer {
+    constructor(private readonly appInsightsClient: ApplicationInsights) {}
+
+    consume(event: IFluidTelemetry) {
+        this.appInsightsClient.trackEvent({
+            name: event.eventName,
+            properties: event,
+        });
+    }
+}
+
+// 3. Next, we'll Create the telemetry config object.
+// Note that we have to obtain the containerId before we can do this.
+const telemetryConfig: TelemetryConfig = {
```

Comment:
Here let's add the instantiation of the appInsightsClient as well as the call to `.load()` or `.initialize()`, or whatever that method was called. So that the client is fully ready when we pass it to the telemetry consumer.